src/main/java/com/microsoft/jenkins/appservice/commands/DockerDeployCommand.java

```diff
@@ -24,10 +24,11 @@
 
 public class DockerDeployCommand extends DockerCommand
         implements ICommand<DockerDeployCommand.IDockerDeployCommandData> {
-    private static final String SETTING_DOCKER_IMAGE = "DOCKER_CUSTOM_IMAGE_NAME";
-    private static final String SETTING_REGISTRY_SERVER = "DOCKER_REGISTRY_SERVER_URL";
-    private static final String SETTING_REGISTRY_USERNAME = "DOCKER_REGISTRY_SERVER_USERNAME";
-    private static final String SETTING_REGISTRY_PASSWORD = "DOCKER_REGISTRY_SERVER_PASSWORD";
+    static final String SETTING_DOCKER_IMAGE = "DOCKER_CUSTOM_IMAGE_NAME";
+    static final String SETTING_REGISTRY_SERVER = "DOCKER_REGISTRY_SERVER_URL";
+    static final String SETTING_REGISTRY_USERNAME = "DOCKER_REGISTRY_SERVER_USERNAME";
+    static final String SETTING_REGISTRY_PASSWORD = "DOCKER_REGISTRY_SERVER_PASSWORD";
```

Comment:
According to another email, these settings are no longer needed now, could you double check?

Reply:
Got it. From my testing, `DOCKER_CUSTOM_IMAGE_NAME` can be safely removed while the other three are still required.